packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -27,6 +28,13 @@ import { v4 as uuidv4 } from 'uuid';
  * @listens pharos-toast-open
  */
 export class PharosToaster extends PharosElement {
+  /**
+   * The elements to return focus to after closing the toast
+   * @attr returnElements
+   */
+  @property({ type: Array, reflect: false })
+  public returnElements: Array<HTMLElement> = [];
+
```

Comment:
Now I'm wondering—is this even a property? It seems like the return elements are communicated only through the custom event detail object. Is that right? If so, this can be removed.

Reply:
ahhhhhh you are right! Updated!